File: solutions/Fox/foo.rb
#!/usr/bin/ruby

require "net/http"
require "uri"

(a, b = Net::HTTP.get(URI.parse("http://directus.darkscience.net:6789")).scan(/([0-9]+) x ([0-9]+)/)[0]) and print /(Correct|Wrong)/.match(Net::HTTP.post_form(URI.parse("http://directus.darkscience.net:6789"), {"answer" => (a.to_i.send '*', b.to_i)}).body)

File: solutions/Fox/foo.pl
#!/usr/bin/perl

use LWP::Simple;

eval "print LWP::UserAgent->new()->post('http://directus.darkscience.net:6789', {'answer' => $1*$2})->decoded_content() =~ /(Correct|Wrong)/g;\n" if get("http://directus.darkscience.net:6789") =~ /([0-9]+) x ([0-9]+)/g